image-rs/src/verity/mod.rs

```diff
@@ -49,7 +48,12 @@ pub fn get_image_name_from_remote(image_url: &str) -> Result<String> {
     Ok(String::from_utf8_lossy(&decoded).to_string())
 }
 
-pub fn cleanup_verity_device(verity_device_name: String) -> Result<()> {
+pub fn create_dmverity_device(verity_options: &str, source_device_path: &Path) -> Result<String> {
+    let parsed_data = DmVerityOption::try_from(verity_options)?;
+    create_verity_device(&parsed_data, source_device_path)
+}
+
+pub fn cleanup_verity_device(verity_device_name: &str) -> Result<()> {
```

Comment:
`destroy_dmverity_device()`? Make it symmetric to `create_dmverity_device()`.

Reply:
Fixed.